src/layouts/RepoList/RepoCard/RepoAvatar/RepoAvatar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { getFirstLetter } from "../../../helpers/parseFunctions";
+import "./RepoAvatar.css";
+
+function RepoAvatar({ avatarLink, repoName }) {
+  return (
+    <div className={"repo-avatar flex-align-center image-container"}>
```

Comment:
можно без фигурных скобок, надо везде будет поправить
```suggestion
    <div className="repo-avatar flex-align-center image-container">
```